### PR TITLE
Added Issue Templates.

### LINKED ISSUE #2: Add Issue Templates
Could you add some relevant issue templates? 

For now, add the below 3 types of issue templates:

1. Bug
2. Feature
3. Others

### PR DESCRIPTION
**Pull request by VividhPandey003**
Fixes Add Issue Templates #2

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+---
+name: Bug report 🐞
+about: Create a report to help us improve
+title: ''
```
Comment (FahimFBA) [inline .github/ISSUE_TEMPLATE/bug_report.md]:
```suggestion
title: '[BUG] Give a suitable title'
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+labels: ''
```
Comment (FahimFBA) [inline .github/ISSUE_TEMPLATE/bug_report.md]:
```suggestion
labels: 'bug'
```

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+---
+name: Feature Request ✨
+about: Suggest an idea for this project
+
```
Comment (FahimFBA) [inline .github/ISSUE_TEMPLATE/feature_request.md]:
```suggestion
title: '[FEAT.] Give a suitable title'
labels: 'enhancement'
assignees: ''
```

File: .github/ISSUE_TEMPLATE/others.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
+---
+name: Other Issue 📝
+about: For any other type of issue that doesn't fit into Bug or Feature Request categories
+
```
Comment (FahimFBA) [inline .github/ISSUE_TEMPLATE/others.md]:
```suggestion
title: "[Other] Give it a suitable title"
labels: ''
assignees: ''
```

**Review by FahimFBA — CHANGES_REQUESTED** [review]

**Comment by FahimFBA** [comment]
Please make the necessary changes.

**Comment by VividhPandey003** [comment]
> Please make the necessary changes.

Commited the suggested changes.

**Review by FahimFBA — APPROVED** [review]

**Comment by FahimFBA** [comment]
Thank you for your contribution. 😃

**Pull request merged**